k8s/README.md

```diff
@@ -48,54 +48,33 @@ The production branch maintains an image that can be released in production with
     -   ./regions/oregon-b/stage-secrets.yaml
     -   ./regions/oregon-b/prod-secrets.yaml
 
-#### Configuration for deploying into a Kops cluster
-
--   Create symbolic links to `kubectl` and `kubeconfig` to use in each region. Note that `kubectl` should match the version of the cluster, or have a version skew of 1 minor version.
```

Comment:
The mention of `kubectl` matching the version of the cluster or having a skew of 1 minor version should probably stay.

Reply:
Not necessarily in this section. Probably somewhere in the beginning of the file would make sense.